src/shared/constants/config/routerPaths.ts

```diff
@@ -41,6 +41,8 @@ export enum PathName {
 
   userProfile = 'profile',
   userProfileSettings = 'settings',
+  userProfileSettingsGeneral = 'settings/general',
+  userProfileSettingsNotifications = 'settings/notifications',
```

Comment:
I've avoided use of multiplePath items in the PathName, there's no problem with it, but better to stay consistent.

```suggestion
  userProfileSettingsGeneral = 'general',
  userProfileSettingsNotifications = 'notifications',
```